webrender/tests/angle_shader_validation.rs

```diff
@@ -56,36 +57,43 @@ const SHADERS: &[Shader] = &[
     },
     Shader {
         name: "ps_text_run",
-        features: PRIM_FEATURES,
+        features: &[ "", "GLYPH_TRANSFORM" ],
     },
     // Brush shaders
     Shader {
         name: "brush_yuv_image",
-        features: &["", "YUV_NV12", "YUV_PLANAR", "YUV_INTERLEAVED", "YUV_NV12,TEXTURE_RECT"],
+        features: &[
+            "",
+            "YUV_NV12",
+            "YUV_PLANAR",
+            "YUV_INTERLEAVED",
+            "TEXTURE_2D,YUV_NV12",
+            "YUV_NV12,ALPHA_PASS",
+        ],
     },
     Shader {
         name: "brush_solid",
-        features: &[],
+        features: BRUSH_FEATURES,
     },
     Shader {
         name: "brush_image",
-        features: &["", "ALPHA_PASS"],
+        features: BRUSH_FEATURES,
     },
     Shader {
         name: "brush_blend",
-        features: &[],
+        features: BRUSH_FEATURES,
     },
     Shader {
         name: "brush_mix_blend",
-        features: &[],
+        features: BRUSH_FEATURES,
     },
     Shader {
         name: "brush_radial_gradient",
-        features: &[ "DITHERING" ],
+        features: &[ "", "DITHERING", "ALPHA_PASS", "DITHERING,ALPHA_PASS" ],
     },
     Shader {
         name: "brush_linear_gradient",
-        features: &[],
+        features: &[ "", "DITHERING", "ALPHA_PASS", "DITHERING,ALPHA_PASS" ],
```

Comment:
dithering wasn't used here before, why is this needed now?

Reply:
I just tought it is a good idea to test both gradient tests with all the possible features.

Reply:
yeah, if the dithering is indeed used, we should test for it :) let's just have a common define for those feature sets